mlflow/server/js/src/experiment-tracking/components/experiment-page/components/runs/ExperimentFilterParams.tsx

```diff
@@ -0,0 +1,90 @@
+import React, { useState, useCallback, useEffect } from 'react';
+import { Theme } from '@emotion/react';
+import { Input } from '@databricks/design-system';
+import { UpdateExperimentSearchFacetsFn } from '../../../../types';
+import { makeCanonicalSortKey } from '../../utils/experimentPage.column-utils';
+import { SearchExperimentRunsFacetsState } from '../../models/SearchExperimentRunsFacetsState';
+import { COLUMN_TYPES } from '../../../../constants';
+
+const FILTER_PARAMS_PLACEHOLDER = 'alpha, lr';
+
+export type ExperimentFilterParamsProps = {
+  updateSearchFacets: UpdateExperimentSearchFacetsFn;
+  searchFacetsState: SearchExperimentRunsFacetsState;
+};
+
+const ExperimentFilterParams = (props: ExperimentFilterParamsProps) => {
```

Comment:
@ahlag Thanks for the PR! I'm curious why we need this component.

Reply:
@harupy 
I am following the structure whereby each bar has its component. Should I merge it in?
https://github.com/mlflow/mlflow/blob/c88d98792cce57c813aba4264732715c4b96d7b5/mlflow/server/js/src/experiment-tracking/components/experiment-page/components/runs/ExperimentViewRunsControls.tsx#L176-L188